site/content/docs/manifest/environment.en.md

```diff
@@ -159,6 +159,33 @@ List of [public AWS Certificate Manager certificate](https://docs.aws.amazon.com
 By attaching public certificates to your load balancer, you can associate your Load Balanced Web Services with a domain name and reach them with HTTPS. 
 See the [Developing/Domains](../developing/domain.en.md#use-domain-in-your-existing-validated-certificates) guide to learn more about how to redeploy services using [`http.alias`](./lb-web-service.en.md#http-alias).
 
+<span class="parent-field">http.public.</span><a id="http-public-access-logs" href="#http-public-access-logs" class="field">`access_logs`</a> <span class="type">Boolean or Map</span>   
+Enable Elastic Load Balancing access logs.   
+If you specify `true`, Copilot will create an S3 bucket to which the Public Load Balancer will store access logs.
+
+```yaml
+http:
+  public:
+    access_logs: true 
+```
+
+You can also specify an existing S3 bucket with a map.
+```yaml
+http:
+ public:
+   access_logs:
+     bucket_name: my-bucket
+     bucket_prefix: access-logs
+```
```

Comment:
What do you think of first giving an example for prefix:
```
You can customize the log prefix:
```yaml
http:
 public:
   access_logs:
     bucket_prefix: access-logs
# ```
```

Then,
```
It is also possible to use your own S3 bucket instead of letting Copilot creates one for you:
```yaml
http:
 public:
   access_logs:
     bucket_name: my-bucket
     bucket_prefix: access-logs
# ```
```

Reply:
Also I just realized the field name for prefix is `bucket_prefix`. My understanding is that the field is really a prefix for the log objects that is being produced. 

Say you have a bucket named "MyBucket". With `bucket_prefix` set to "myprefix", the logs will be created in `MyBucket/myprefix/AWSLogs/accountID/...`. Without `bucket_prefix` it'd be `MyBucket/AWSLogs/accountID...`. So it's not a prefix to the bucket, but a prefix to the log objects.

Is this understanding correct? If so, would an alternative such as `log_prefix` be a more accurate name?